index.js

```
@@ -7,24 +8,33 @@ const commit = require('./lib/lifecycles/commit')
 const tag = require('./lib/lifecycles/tag')
 
 module.exports = function standardVersion (argv) {
-  var pkg
+  let pkg
   bump.pkgFiles.forEach((filename) => {
     if (pkg) return
     var pkgPath = path.resolve(process.cwd(), filename)
     try {
       pkg = require(pkgPath)
     } catch (err) {}
   })
-  if (!pkg) {
-    return Promise.reject(new Error('no package file found'))
-  }
-  var newVersion = pkg.version
-  var defaults = require('./defaults')
-  var args = Object.assign({}, defaults, argv)
+  let newVersion
+  let defaults = require('./defaults')
+  let args = Object.assign({}, defaults, argv)
 
   return Promise.resolve()
     .then(() => {
-      return bump(args, pkg)
+      if (!pkg && args.gitTagFallback) {
+        return latestSemverTag()
+      } else if (!pkg) {
+        return Promise.reject(new Error('no package file found'))
```

Instead of returning a rejected promise, could you just throw the error here?